Automattic/wp-calypso
# E2E: Rotate store sandbox cookie

Last rotation: https://github.com/Automattic/wp-calypso/pull/81190

## Proposed Changes

Rotate the store sandbox cookie value due to an employee departing the company. 

Accompanying diff: D128245-code 

> [!Caution]
> The backend diff needs to be merged first!

## Testing Instructions

Decrypt the secrets (PCYsg-vnR-p2) and ensure the `storeSandboxCookieValue` matches the one in the accompanying backend diff.

> [!Note]
> Since all the tests that use the sandbox cookie are a part of the pre-release suite, changes in this PR will only affect those tests once merged.
